src/Momento.Sdk/Responses/CacheDeleteResponse.cs

```diff
@@ -2,7 +2,7 @@
 
 using Momento.Sdk.Exceptions;
 
-public class CacheDeleteResponse
+public abstract class CacheDeleteResponse
```

Comment:
Why the abstract modifier?

Reply:
All of the parent response classes are supposed to be abstract so they can't be instantiated. I just missed this one on the first go.

Reply:
Thanks for the context!